libs/sync/diff.go

```diff
@@ -15,6 +19,71 @@ func (d diff) IsEmpty() bool {
 	return len(d.put) == 0 && len(d.delete) == 0
 }
 
+// Compute operations required to make files in WSFS reflect current local files.
+// Takes into account changes since the last sync iteration.
+func computeDiff(target *SnapshotState, current *SnapshotState) diff {
+	d := &diff{
+		delete: make([]string, 0),
+		rmdir:  make([]string, 0),
+		mkdir:  make([]string, 0),
+		put:    make([]string, 0),
+	}
+	d.addRemovedFiles(target, current)
+	d.addFilesWithRemoteNameChanged(target, current)
+	d.addNewFiles(target, current)
+	d.addUpdatedFiles(target, current)
```

Comment:
Naming is different between this and the other functions (target/current vs after/before).

Reply:
done